src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -21,7 +21,7 @@ package scoverage
 
 import java.nio.file.{Path, Paths}
 
-import com.mwz.sonar.scala.scoverage.ScoverageSensorInternal._
+import com.mwz.sonar.scala.scoverage.ScoverageSensor._
```

Comment:
Could you please move this import to the top of the class, and add the scalastyle ignore comment?